pyproject.toml

```diff
@@ -30,6 +30,7 @@ dependencies = [
     "rs-parsepatch",
     "sentry-sdk",
     "setuptools-scm",
+    "simple-github",
```

Comment:
You will need to re-generate `requirements.txt` to include this in the Docker container.